tests/_testrunner.livecodescript

```diff
@@ -208,19 +208,38 @@ private command TestLoadIDE
    return tAnalysis
 end TestLoadIDE
 
+private command appendToList pElt, @xList
+   put pElt into xList[the number of elements in xList + 1]
+end appendToList
+
+private function getAllTestFileNames
+   local tIDETests, tIDESupportTests
+   put TestGetIDERepositoryPath() & "/tests" into tIDETests
+   put TestGetEngineRepositoryPath() & "/tests/ide-support" into tIDESupportTests
```

Comment:
Could just bite the bullet and create a separate text file that lists test sources. 😉